yarn.lock

```diff
@@ -2,6 +2,13 @@
 # yarn lockfile v1
 
 
+"@agoric/assert@^0.3.1":
```

Comment:
Hmm, most of the changes in this yarn.lock shouldn't be happening. Before we merge this, can we make sure we've gotten the package dependencies right? For instance, an old version of Nat is being installed, and multiple Agoric packages are being added to the root yarn.lock even though they are already widely used.